project/src/components/places-list/places-list.tsx

```diff
@@ -1,13 +1,20 @@
+import { Offer } from '../../types/offer';
 import PlaceCard from '../place-card/place-card';
+import {useState} from 'react';
 
-type Cards = {
-  list: number[]
-};
+type Offers = {
+  list: Offer[]
+}
+
+function PlacesList({list}: Offers): JSX.Element {
+  const [activeCard, setActiveCard] = useState({});
+  const onMouseEnterHandler = (offer: Offer) => {
+    setActiveCard(offer);
+  };
 
-function PlacesList({list}: Cards): JSX.Element {
   return (
     <div className="cities__places-list places__list tabs__content">
-      {list.map((card: number) => <PlaceCard key={card}/>)}
+      {list.map((card: Offer) => <PlaceCard key={card.id} offer={card} onMouseEnter={onMouseEnterHandler} />)}
```

Comment:
Когда передаем как пропс называем on + . Когда описываем обработчик handle +
```suggestion
      {list.map((card: Offer) => <PlaceCard key={card.id} offer={card} onMouseEnter={handleMouseMove} />)}
```